vmb/templates/matrimony/emails/matches.html

```diff
@@ -138,6 +138,17 @@ <h4>Medical Details</h4>
                   </p>
                 </td>
               </tr>
+              <tr>
```

Comment:
You don't need a `form` here, just a button type link should do the trick. The link/button when clicked should open the link in a new tab.